### PR TITLE
[cmake] fix package names that find_package is equal to find_package_handle_standard_args

### PR DESCRIPTION
**Pull request by Rechi**
## Description
The package name in `find_package_handle_standard_args` must be the exact same as in `find_package` otherwise `REQUIRED` has no effect.

## Motivation and Context
@notspiff pointed out that this was wrong for rapidjson.
That is the change for the other packages.
@wsnipex fyi

## How Has This Been Tested?
reconfigured with cmake after deleting `CMakeCache.txt`

## Types of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -107,7 +107,7 @@ set(required_deps Cdio
                   FFMPEG
                   Fmt
                   FreeType
-                  Fribidi
+                  FriBidi
```
Comment (MilhouseVH) [inline CMakeLists.txt]: I think you now need to rename

https://github.com/xbmc/xbmc/blob/master/cmake/modules/FindFribidi.cmake

to `cmake/modules/FindFriBidi.cmake`

on Linux.

**Comment by Rechi** [comment]
@MilhouseVH thanks, updated

**Comment by Rechi** [comment]
@hudokkow can you have a look?

**Review by wsnipex — APPROVED** [review]

**Comment by wsnipex** [comment]
jenkins build this please

**Comment by garbear** [comment]
On windows, with this PR, I'm getting a cmake error:

```
-- Could NOT find PkgConfig (missing:  PKG_CONFIG_EXECUTABLE)
-- Found EXPAT: C:/Users/Garrett/Documents/kodi/project/BuildDependencies/lib/expat.lib (found version "2.2.0")
CMake Error at C:/Program Files/CMake/share/cmake-3.8/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find Cdio (missing: CDIO_LIBRARY)
Call Stack (most recent call first):
  C:/Program Files/CMake/share/cmake-3.8/Modules/FindPackageHandleStandardArgs.cmake:377 (_FPHSA_FAILURE_MESSAGE)
  cmake/modules/FindCdio.cmake:29 (find_package_handle_standard_args)
  cmake/scripts/common/Macros.cmake:315 (find_package)
  CMakeLists.txt:169 (core_require_dep)


-- Configuring incomplete, errors occurred!
```

reverting back to master fixes this error.

**Comment by Rechi** [comment]
@garbear have you tried to remove `CMakeCache.txt` or removing the build folder completely and afterwards reconfigured?
The first commit of the PR should fix the find issue on windows.

**Comment by garbear** [comment]
just tested windows again, no failures

**Comment by tudstudent** [comment]
Rechi, just tested on Ubuntu 16.04 X64 and it jams at rapidjson.
It is now recognizing FMT.

`-- Found FriBidi: /usr/lib/x86_64-linux-gnu/libfribidi.so (found version "0.19.7")
-- Found PCRE: /usr/lib/x86_64-linux-gnu/libpcrecpp.so (found version "8.38")
CMake Error at /usr/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:148 (message):
  Could NOT find RapidJSON (missing: RapidJSON_VERSION)
Call Stack (most recent call first):
  /usr/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:388 (_FPHSA_FAILURE_MESSAGE)
  cmake/modules/FindRapidJSON.cmake:27 (find_package_handle_standard_args)
  cmake/scripts/common/Macros.cmake:315 (find_package)
  CMakeLists.txt:169 (core_require_dep)


-- Configuring incomplete, errors occurred!
See also "/home/htpc01/src/Kodi/kodi/kodi-build/CMakeFiles/CMakeOutput.log".
See also "/home/htpc01/src/Kodi/kodi/kodi-build/CMakeFiles/CMakeError.log".`

**Comment by Rechi** [comment]
@tudstudent rapidjson failure isn't related with this PR as I haven't changed anything related to it.

**Pull request merged**